deploy/runtime/startup.lk

```diff
@@ -559,9 +559,10 @@ function setup_fuel_cell_pages_fom()
 			'Battery Dispatch Front of Meter',
 			'Fuel Cell Dispatch Manual']], {'sidebar'='Dispatch', 'help'='fuelcell_dispatch'});*/
 	addpage( [[ {'name'='Battery Dispatch Automated FOM', 'caption'='Automated'} ], [ {'name'='Battery Dispatch Custom Time Series', 'caption'='Custom Time Series'} ], 
-		['Battery Dispatch Empty'],[{'name'='Battery Dispatch PV Smoothing', 'caption'='PV Smoothing'}]],
+		[{'name'='Fuel Cell Dispatch Manual', 'caption'='Manual'}],
```

Comment:
So @cpaulgilman and I have discussed this issue. The battery manual dispatch page for fuel cell also includes the controls for fuel cell manual dispatch. So if the user selects Manual Dispatch for fuel cell dispatch but not for battery dispatch, they will not be able to see the controls for manual fuel cell dispatch. This was why the Fuel Cell Dispatch Manual was fixed as a header page.